doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+
+Note that we need to compute the loss gradient through the model. If we have access to the internals, we can do this directly. Otherwise, we need to use numerical differentiation at a high computational cost due to the extra model calls we need to make.
+
+**Pros**
+- Both a black and white box method
+- We obtain more interpretable results using the autoencoder loss as $\delta$ will be in the data distribution
+
+**Cons**
+- Finding non-informative feature values to add or take away from an instance is often not trivial, and domain knowledge is essential
+- Need to tune hyperparameters $\beta$ and $\gamma$
+- The insight doesn't tell us anything about the coverage of the pertinent positive
+- If using the autoencoder loss, then we need access to the original dataset
+
+### Local Feature Attribution
+
+Local feature attribution asks how each feature in a given instance contributes to its prediction. In the case of an image, this would highlight those pixels that make the model provide the output it does. Note that this differs subtly from Local Necessary Features, which find the minimum subset of features required to give a prediction. Local feature attribution instead assigns a score to each feature.
+
+__TODO__:
+- picture showing above.
+
+A good example use of local feature attribution is to detect that a classifier trained on images is focusing on the correct features of an image to infer the class. Suppose you have a model trained to classify breeds of dogs. You want to check that it focuses on the correct features of the dog in making its prediction. Suppose you compute the feature attribution of a picture of a husky and discover that the model is only focusing on the snowy backdrop to the husky, then you know two things. All the images of huskies in your dataset overwhelmingly have snowy backdrops, and also that the model will fail to generalize. It will potentially incorrectly classify other dog breeds with snowy backdrops as huskies and fail to recognize huskies that aren't in snowy locations.
+
+Each of the following methods defines local feature attribution slightly differently. In both, however, we assign attribution values to each feature to indicate how significant those features were in making the model prediction.
+
+Let $f:\mathbb{R}^n \rightarrow \mathbb{R}$. $f$ might be a regression, a single component of a multi regression or a probability of a class in a classification model. If $x=(x_1,... ,x_n) \in \mathbb{R}^n$ then an attribution of the prediction at input $x$ is a vector $a=(a_1,... ,a_n) \in \mathbb{R}^n$ where $a_i$ is the contribution of $x_i$ to the prediction $f(x)$.
```

Comment:
regression --> regression model
multi regression --> multi-output regression (multi regression can be confused with multiple regression which means many features)